### ISSUE TITLE
[EVENTOS] Falta de clareza do scroll lateral no Painel de controle

### ISSUE THREAD

**Issue by emillecatarine** [LabCDBr/gestao#224]

**Comment by RaniGub** [comment]
@emillecatarine se não me engano, existe situações que o Mapas apresenta uma seta para o lado direito e esquerdo por mais que não tenham novos registro para serem apresentados, você chegou a se deparar com isso?

**Comment by emillecatarine** [comment]
sim, @RaniGub ! bem observado, vou criar uma issue para ajustarmos isso também.
